### DISCUSSION TITLE
How to use Cypress Component Testing in Docker

### DISCUSSION THREAD

**Discussion by Develliot** [cypress-io/cypress#16009] [Component Testing] [answered]
https://www.cypress.io/blog/2019/05/02/run-cypress-with-a-single-docker-command/

in docker compose:

image: cypress/included:7.1.0
entrypoint: npx cypress run-ct

It seems to be running, but even though tests are failing it's not displaying the failures for some reason, I probably need to add something to this entry point but I haven't got a clue.

**Comment by lmiller1990** [comment, marked as answer]
This should definitely work, probably just a matter of digging in to see what's going on. Sorry I don't have an answer for you right now, I can look into this in the next week or two.

**Reply by skyred** [reply]
It works. Here is the command I used at the root of your component project

```docker
docker run -it --rm --privileged -v $PWD:/e2e -v /tmp/.X11-unix:/tmp/.X11-unix -w /e2e -e DISPLAY=unix$DISPLAY --entrypoint cypress cypress/included:9.6.0 open-ct --project .
```
